webapp/components/PostCard/PostCard.vue

```diff
@@ -167,4 +179,8 @@ export default {
     text-indent: -999999px;
   }
 }
+
+.post--pinned {
+  border: 1px solid $color-warning;
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-21T08:47:16Z" title="Monday, October 21st 2019, 10:47:16 am +02:00">Oct 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -167,4 +179,8 @@ export default {
     text-indent: -999999px;
   }
 }
+
+.post--target {
+  border: 1px solid $color-primary;
```


This breaks the design unfortunately and therefore I don’t like so much … 😬